src/Bids/Lebensohl.hs

```diff
@@ -159,3 +166,46 @@ b1No2D3S = gfWithSuit_ T.Spades T.Diamonds
 
 b1No2H3S :: Action
 b1No2H3S = gfWithSuit_ T.Spades T.Hearts
+
+
+-- This is the meat of lebensohl!
+bLebensohl2N :: Action
+bLebensohl2N = do
```

Comment:
No, make separate ones for each overcall, because the `alternatives` to include are different for each one.